package.json

```diff
@@ -64,7 +65,10 @@
     "release": "standard-version -a",
     "next-release": "standard-version --scripts.prebump=./build/next-version.js --skip.commit=true --skip.tag=true",
     "sri-update": "grunt build && node build/sri-update && git add sri-history.json",
-    "fmt": "prettier --write  *.js, **/*.ts '{build,doc,lib,test}/**/*.js'"
+    "fmt": "prettier --write  *.js, **/*.ts '{bin,build,doc,lib,test}/**/*.js'"
+  },
+  "bin": {
```

Comment:
Why are you adding this?

Reply:
Since the binary is using development dependencies, this won't work.

Please remove `"bin": { ... }`.